openedx/core/djangoapps/catalog/utils.py

```diff
@@ -239,6 +242,47 @@ def get_course_runs_for_course(course_uuid):
     else:
         return []
 
+def get_visible_course_runs_for_course(entitlement, course_uuid):
```

Comment:
The entitlement has the course_uuid in it, you should only need it as a parameter.